ts/nni_manager/training_service/reusable/environments/heterogenousEnvironmentService.ts

```diff
@@ -76,7 +95,7 @@ export class HeteroGenousEnvironmentService extends EnvironmentService {
     }
 
     public async startEnvironment(environment: EnvironmentInformation): Promise<void> {
-        const number = randomSelect([0, 1]);
+        const number = randomSelect([0, 1, 2, 3]);
```

Comment:
Should it to be in GPU scheduler?

Reply:
No. This is used to initialize environment, for example choose to submit a environment on AML or PAI. After assign a environment, then gpu scheduler is used for trials to assign environment.

Reply:
It likes two level gpu scheduler with this design.

Reply:
refactored code, mode environment service scheduler to trialDispatcher.ts.